black

```diff
@@ -0,0 +1,9 @@
+[tool.black]
```

Comment:
https://github.com/NRLMMD-GEOIPS/geoips/blob/main/.config/black

This matches the geoips default branch black config. Not sure how we want to consolidate these.